tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
-	"extends": "./node_modules/@amaurym/tsconfig/tsconfig",
-	"compilerOptions": {
-		"declaration": true,
-		"module": "commonjs",
-		"outDir": "./lib",
-		"skipLibCheck": true,
-		"target": "es5"
-	}
+  "extends": "./node_modules/@amaurym/tsconfig/tsconfig",
```

Comment:
actually, we should also unroll that config above, and put it explicity in this file, so remove the dependency on @amaurym/tsconfig

Reply:
unrolled ✅